src/peerbook/libp2p_peer.erl

```diff
@@ -48,9 +48,16 @@ from_map(Map, SigFun) ->
     Assocs = lists:map(fun({Type, AssocEntries}) ->
                                {Type, #libp2p_association_list_pb{associations=AssocEntries}}
                        end, maps:get(associations, Map, [])),
+    Connected0 = maps:get(connected, Map, []),
+    MaxConns = application:get_env(libp2p, max_connected_peers, 8),
```

Comment:
This is not really limiting the number of connected peers is it? Naming is hard :-/ Maybe call it max_enoded_peers or max_gossiped_peer_connnections